src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/client/AssessRisksAndNeedsApiClient.kt

```diff
@@ -0,0 +1,18 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.client
```

Comment:
This commit message seems to have trailed off. FWIW when a commit subject goes past the 50 character limit, I tend to like to shorten the subject and add more context in the description.

Reply:
Have reworded this and the similar one for tier